rules/windows/process_creation/proc_creation_win_notepad_local_passwd_discovery.yml

```diff
@@ -0,0 +1,33 @@
+title: Notepad Password Files Discovery
+id: 3b4e950b-a3ea-44d3-877e-432071990709
+status: experimental
+description: Detects the execution of Notepad to open a file that has the string "password" which may indicate unauthorized access to credentials or suspicious activity.
+example_cmdline:
+    - 'NOTEPAD.EXE C:\Shares\Passwords\admin.txt'
+    - 'NOTEPAD.EXE C:\Users\Public\passwords.csv'
```

Comment:
Unfortunately, we do not allow extra fields in this repo. I'll add them to the PR body 
```suggestion
```